### PR TITLE
feat: export meta from useWidget

### PR DESCRIPTION
**Pull request by samobasquiat**
# Summary

In the dapp we call meta as well & delaying in fetching meta caueses some issues. We decieded to get meta from useWidget hook.

Fixes # (issue)

# How did you test this change?

there is a WIP pr in dapp side:
https://github.com/rango-exchange/app/pull/17
with linking it can be tested.

# Checklist:

- [x] I have performed a self-review of my code

### CODE REVIEW

File: widget/embedded/src/containers/WidgetInfo/WidgetInfo.helpers.ts [hunk 1]
```diff
@@ -11,9 +13,14 @@ import { getPendingSwaps } from '../../utils/queue';
 
 export class WidgetHistory {
   private manager: Manager | undefined;
+  private retrySwap: (pendingSwap: PendingSwap, meta: Meta) => void;
 
-  constructor(manager: Manager | undefined) {
+  constructor(
+    manager: Manager | undefined,
+    retrySwap: (pendingSwap: PendingSwap, meta: Meta) => void
```
Comment (yeager-eren) [inline widget/embedded/src/containers/WidgetInfo/WidgetInfo.helpers.ts:20]: I think make the second parameters to a more general object would be better.
`actions: { retrySwap: .... }`. 
lets us to add/remove more easily in futute.

File: widget/embedded/src/containers/Wallets/Wallets.tsx [hunk 2]
```diff
@@ -40,6 +41,10 @@ function Main(props: PropsWithChildren<PropTypes>) {
   const onConnectWalletHandler = useRef<OnConnectHandler>();
   useSyncStoresWithConfig();
 
+  useEffect(() => {
+    void fetchMeta().catch();
```
Comment (yeager-eren) [inline widget/embedded/src/containers/Wallets/Wallets.tsx:45]:
```suggestion
    void fetchMeta().catch(console/log);
```
I know you just moved the code, but completely ignoring the error will make frustrations in future. we should see the error in console at least.

File: widget/embedded/src/containers/WidgetInfo/WidgetInfo.helpers.ts (hunk 1 above)
Comment (samobasquiat) [reply widget/embedded/src/containers/WidgetInfo/WidgetInfo.helpers.ts]: fixed.

File: widget/embedded/src/containers/Wallets/Wallets.tsx (hunk 2 above)
Comment (samobasquiat) [reply widget/embedded/src/containers/Wallets/Wallets.tsx]: fixed.

**Pull request merged**